src/main/java/spoon/reflect/visitor/ImportScannerImpl.java

```diff
@@ -63,12 +63,10 @@
 
 	@Override
 	public <T> void visitCtInvocation(CtInvocation<T> invocation) {
-		// For a ctinvocation, we don't have to import declaring type
 		enter(invocation);
 		scan(invocation.getAnnotations());
 		scanReferences(invocation.getTypeCasts());
 		scan(invocation.getTarget());
-		scan(invocation.getExecutable());
```

Comment:
I already try to remove this line but some time this line is required.
I don't remember in which case (may be in static call).